.github/workflows/js-tests.yml

```diff
@@ -63,7 +63,10 @@ jobs:
       run: |
         make base-requirements
 
-    - uses: c-hive/gha-npm-cache@v1
```

Comment:
You removed the cache action, but upper the cache attribute is not added in the `actions/setup-node@v4`
https://github.com/openedx/edx-platform/pull/36502/files